packages/nx/src/adapter/ngcli-adapter.ts

```diff
@@ -599,9 +607,7 @@ export class NxScopeHostUsedForWrappedSchematics extends NxScopedHost {
         // we have to add them into the file.
         const createdProjectFiles = findCreatedProjects(this.host);
         const deletedProjectFiles = findDeletedProjects(this.host);
-        const nxJsonInTree = nxJsonChange
-          ? parseJson(nxJsonChange.content.toString())
-          : parseJson(this.host.read('nx.json').toString());
+        const nxJsonInTree = readNxJson(this.host);
```

Comment:
@leosvelperez and I discussed this morning, since this.host **_is_** our tree, not angular's, and findMatchingChange **_is_** using our tree, we can just use the devkit util.